bin/rename

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+CURRENT_NAME="PhoenixStarter"
+CURRENT_OTP="phoenix_starter"
+
+echo "Please enter new name (current: $CURRENT_NAME)"
+read NEW_NAME
+
+echo "Please enter new OTP name (current: $CURRENT_OTP)"
+read NEW_OTP
+
+# Ignore errors
```

Comment:
Simple comment explaining why this is needed would be great, don't you think?

Reply:
What part? Of ignoring the errors?

Reply:
Yes. Why do we need to ignore the errors? Is this temporary or is it a requirement?

Just something that explains why those couple of lines are there, because it is not apparent and I would prefer to not have to dive into the Linux Manual to figure out 😅

Reply:
If for some reason you want to ignore error you can you set + and set -e

Reply:
Doesn't seem to work tho

Reply:
```
set +e 
your code that can exit with value diferent than 0 
set -e 

normal code 
```
This doesn't work ?

Reply:
I want to hide the error messages. Sed gives a file not found error somehow, but still manages to find and replace every single file I want.

Reply:
But still i can at least omit the `|| true ` part with that

Reply:
`sed` is giving a file not found error maybe because you are passing all the files that are in the git repo to it, including images and frontend assets.

You don't need to omit the error if you limit the files you pass to `sed` to the ones you actually want to do the replace.

Something like this would probably work:
`git ls-tree -r master --name-only | grep -E "(README.md|mix.exs|config/|lib/|priv/|test/)" | xargs sed -i '' -e "s/$CURRENT_NAME/$NEW_NAME/g"
`

Reply:
Was an error with an empty line being fed by xargs into sed. `-r` option fixed it (it's useless on mac as it does this by default but at least doesn't crash).